packages/core/src/api/parsers/html/parseHTML.test.ts

```diff
@@ -33,16 +33,15 @@ async function parseHTMLAndCompareSnapshots(
 
   (window as any).__TEST_OPTIONS.mockID = 0; // reset id counter
   const htmlNode = nestedListsToBlockNoteStructure(html);
-  const tt = editor._tiptapEditor;
 
   const slice = view.__parseFromClipboard(
-    tt.view,
+    editor.prosemirrorView,
```

Comment:
I get why we use `editor.dispatch` instead of `tt.view.dispatch`, but what is this for? Isn't `editor.prosemirrorView` just a getter for `tt.view`?

Reply:
true, was just refactoring references to `tt`. We could have kept the original here